.codegen/tests/cmd/workspace/test.toml.tmpl

```diff
@@ -0,0 +1,101 @@
+# Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+{{- $excludes :=
+	list
+		"command-execution"
+		"statement-execution"
+		"dbfs"
+		"dbsql-permissions"
+		"account-access-control-proxy"
+		"files"
+        "dashboards"
+        "queries-legacy"
+}}
+
+Local = true
+Cloud = false
+
+[[Repls]]
+Old="Date: .*"
+New="Date: (redacted)"
+
+[[Repls]]
+Old="Traceparent: .*"
+New="Traceparent: (redacted)"
```

Comment:
This part is common across all templates, can we extract it into non-templated test.toml in parent directory?

Reply:
Good point, done